src/main/java/core/basesyntax/Lottery.java

```diff
@@ -0,0 +1,9 @@
+package core.basesyntax;
+
+public class Lottery {
+
+    public static void getRandomBall() {
+        Ball ball = new Ball();
+        System.out.println(ball.getBall());
```

Comment:
Please pay attention to what happens here:
1. You create an instance of an ball.
2. You try to get ball... out of the ball.

What should happen here is:
1. Prepare random number and color
2. Create and return an instance of a ball with prepared values (passed to constructor).